docker/docker-compose.yml

```diff
@@ -87,14 +87,15 @@ services:
       - ENTITY_SERVICE_HOST_CONFIG=hypertrace
       - ENTITY_SERVICE_PORT_CONFIG=9001
       - KAFKA_SINK_TOPIC=enriched-structured-traces
+      - PRE_CREATE_TOPICS=true
     volumes: *default-log-config
     depends_on:
       kafka-zookeeper:
         condition: service_healthy
       schema-registry:
         condition: service_healthy
       hypertrace:
-        condition: service_healthy
+        condition: service_started
```

Comment:
make a comment? I think last time schema-registry was scapegoated not hypertrace